app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainFragment.kt

```diff
@@ -30,7 +32,7 @@ class MainFragment : Fragment() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
+        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
```

Comment:
Changed this to the newest form, the second link explains the ViewModelProvider: 
1) https://developer.android.com/reference/androidx/lifecycle/ViewModelProviders
2) https://developer.android.com/reference/androidx/lifecycle/ViewModelProvider